recipes/scikit-bio/meta.yaml

```diff
@@ -0,0 +1,98 @@
+{% set version = "0.4.2" %}
+
+package:
+  name: scikit-bio
+  version: {{ version }}
+
+source:
+  fn: scikit-bio-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-bio/scikit-bio-{{ version }}.tar.gz
+  md5: bbbe704410c59083e9eb65fa9d01a50d
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - bz2file >=0.98
+    - lockfile >=0.10.2
+    - cachecontrol >=0.11.5
+    - contextlib2 >=0.4.0
+    - decorator >=3.4.2
+    - future >=0.14.3
+    - ipython >=3.2.0
+    - matplotlib >=1.4.3
+    - natsort >=4.0.3
+    - numpy >=1.9.2
+    - pandas 0.17.*
+    - scipy >=0.15.1
+    - six >=1.9.0
+    - nose >=1.3.7
+
+  run:
+    - python
+    - bz2file >=0.98
+    - lockfile >=0.10.2
+    - cachecontrol >=0.11.5
+    - contextlib2 >=0.4.0
+    - decorator >=3.4.2
+    - future >=0.14.3
+    - ipython >=3.2.0
+    - matplotlib >=1.4.3
+    - natsort >=4.0.3
+    - numpy >=1.9.2
+    - pandas 0.17.*
+    - scipy >=0.15.1
+    - six >=1.9.0
+    - nose >=1.3.7
+
+test:
+  imports:
+    - skbio
+    - skbio.alignment
+    - skbio.alignment._lib
+    - skbio.alignment.tests
+    - skbio.diversity
+    - skbio.diversity.alpha
+    - skbio.diversity.alpha.tests
+    - skbio.diversity.beta
+    - skbio.diversity.beta.tests
+    - skbio.diversity.tests
+    - skbio.io
+    - skbio.io.format
+    - skbio.io.format.tests
+    - skbio.io.tests
+    - skbio.sequence
+    - skbio.sequence.tests
+    - skbio.stats
+    - skbio.stats.distance
+    - skbio.stats.distance.tests
+    - skbio.stats.evolve
+    - skbio.stats.evolve.tests
+    - skbio.stats.ordination
+    - skbio.stats.ordination.tests
+    - skbio.stats.tests
+    - skbio.tests
+    - skbio.tree
+    - skbio.tree.tests
+    - skbio.util
+    - skbio.util.tests
+  requires:
+    - nose
+    - httpretty
+  commands:
+    - python -m skbio.test  # [no linux]
```

Comment:
Ah, I see, fixed.

On (Jul-25-16| 9:34), jakirkham wrote:
> > +    - skbio.stats.distance.tests
> > +    - skbio.stats.evolve
> > +    - skbio.stats.evolve.tests
> > +    - skbio.stats.ordination
> > +    - skbio.stats.ordination.tests
> > +    - skbio.stats.tests
> > +    - skbio.tests
> > +    - skbio.tree
> > +    - skbio.tree.tests
> > +    - skbio.util
> > +    - skbio.util.tests
> > +  requires:
> > +    - nose
> > +    - httpretty
> > +  commands:
> > +    - python -m skbio.test  # [no linux]
> 
> Sorry! I should have said `not linux`.
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/conda-forge/staged-recipes/pull/1071/files/6d888adae42bb54e4e6a194a36e335068ff9bc06#r72097869